docs/manual/layout/site-structure/pages-as-central-elements.en.md

```diff
@@ -59,14 +59,14 @@ There are eight different page types with different features in Contao, each of
-| 403 Access denied | This page is called if a member is logged in but does not have the necessary access rights to access a protected page. You can choose to use the page as a regular page and display a hint or automatically redirect the visitor to another page. <sup>1</sup>                                      |
-| 404 Page not found | This page is called when a visitor requests a page that does not exist. You can choose to use the page as a regular page and include a sitemap or automatically redirect the user to another page.                                                                                                  |
+| Regular page | Regular pages are pages that output content. A regular page is similar to a static HTML file that you upload to your server and request in your browser.                                                                                                                                     |
+| Internal forwarding | A page of this type redirects visitors to another page in the page structure. The target page must be accessible under the same domain as the redirection page, otherwise an external redirection must be used.                                                                                          |
+| External forwarding | A page of this type redirects visitors to an external page. This can be a page outside your server or a page within the Contao page structure but under a different domain than the redirect page.                                                                                                       |
+| Starting point of a website | A page of this type is the starting point of a website within the page structure. A Contao instance can manage multiple websites, which can, for example. have different languages or run completely independently under different domains (multi-domain operation). |
+| Log off | A page of this type creates a logout link for a protected area. After logging out, you can redirect visitors to any page or to the last page they visited.                                                                                                                                               |
+| 401 Not authenticated | {{< version-tag "4.6" >}} This page appears when a visitor is not logged in and is therefore not allowed to access a protected page. You can choose to use the page as a regular page and display a hint or automatically redirect the visitor to the login page. <sup>1</sup>                  |
+| 403 Access denied | This page appears if a user is logged in, but does not have the necessary access rights to access a protected page. You can choose to use the page as a regular page and display a hint or automatically redirect the visitor to another page. <sup>1</sup>                                      |
```

Comment:
Front end users in Contao are called "members" so may be we should keep the terminology here.